st2rbac_backend/resolvers.py

```diff
@@ -636,14 +721,55 @@ class KeyValuePermissionsResolver(PermissionsResolver):
     """
 
     resource_type = ResourceType.KEY_VALUE_PAIR
+    view_grant_permission_types = [
+        PermissionType.KEY_VALUE_VIEW,
+        PermissionType.KEY_VALUE_SET,
+        PermissionType.KEY_VALUE_DELETE,
+        PermissionType.KEY_VALUE_LIST,
+        PermissionType.KEY_VALUE_ALL,
+    ]
```

Comment:
Do we need to override user_has_resource_api_permission, abstract method for PermissionResolver at  https://github.com/StackStorm/st2-rbac-backend/blob/master/st2rbac_backend/resolvers.py#L84?